web/app/.eslintrc

```diff
@@ -0,0 +1,52 @@
+{
+  "plugins": [
+    "eslint-plugin-react", // https://github.com/yannickcr/eslint-plugin-react
+    "eslint-plugin-promise"
+  ],
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
+  "env": {
+    "browser": true,
+    "es6": true
+  },
+  "extends" : ["eslint:recommended", "plugin:react/recommended"],
+  "rules": {
+    "arrow-parens": [1, "as-needed"],
+    "comma-dangle": [0], // babel removes dangling commas for us
+    "eqeqeq": [2, "smart"],
+    "indent": [2, 2],
+    "jsx-quotes": 1,
+    "no-console": [1, { allow: ["warn", "error"] }], // minor offense
```

Comment:
nit: for consistency, we should double quote all the property names in this file